config/default.conf

```diff
@@ -1,42 +1,63 @@
 [global]
 
-server.socket_host: '0.0.0.0'
-server.socket_port: 81
-
-response.headers.server: "Nothing to see here ... move along :)"
-
-tools.jinja.on: False
-tools.caching.on: False
-tools.isAuthorised.on: False
-
-log.access_file : None
-log.error_file  : 'logs/error.log'
-
-
-email: "EMAIL"
-
-#headnode: "https://HEADNODE/RPC2"
-
-# Sets default values for users set to unlimited
-unlimited-vm: "Unlimited"
-unlimited-cpu: 64
-unlimited-mem: 61440
-unlimited-sys: 61440
-
-auth.session_expire_secs: 86400
-auth.ldap_address: "ldap://LDAP"
-auth.ldap_basedn: ""
-
-goodwords: "/usr/share/dict/words"
-# you must provide this file yourself
-badwords: "/var/www/frontend/badwords"
-
-# websockify config for NoVNC
-wshostname: "HOSTNAME"
-wsport: 29876
-#wscert: "/etc/httpd/ssl/CERT"
-#wskey: "/etc/httpd/ssl/KEY"
-wstokendir: "/var/www/frontend/tokens/"
-
-# set to True if using Apache
-wsgi_enabled: True
+# Auth
+auth.ldap_address = "ldap://LDAP"
+auth.ldap_basedn = "dc=FED,dc=CCLRC,dc=AC,dc=UK"
```

Comment:
Is this correct? Shouldn't the keys be different and capitalised?

Also, should it not be a generic version in this repository?